src/payment/payment-request-transformer.ts

```diff
@@ -35,7 +35,8 @@ export default class PaymentRequestTransformer {
         const orderMeta = checkoutState.order.getOrderMeta();
         const internalCustomer = customer && billingAddress && mapToInternalCustomer(customer, billingAddress);
 
-        const authToken = instrumentMeta && payment.paymentData && isVaultedInstrument(payment.paymentData) ?
+        const authToken = instrumentMeta && payment.paymentData &&
+        (isVaultedInstrument(payment.paymentData) || isFormattedInstrument(payment.paymentData)) ?
```

Comment:
Just some indenting here.

```suggestion
            (isVaultedInstrument(payment.paymentData) || isFormattedInstrument(payment.paymentData)) ?
```